feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -180,15 +181,15 @@ class BillingWrapper(
         executeRequestOnUIThread {
             val params = BillingFlowParams.newBuilder()
                 .setSkuDetails(productDetails.skuDetails)
-                // Causing issues with downgrades/upgrades https://issuetracker.google.com/issues/155005449
-                // .setObfuscatedAccountId(appUserID.sha256())
                 .apply {
                     replaceSkuInfo?.apply {
                         setOldSku(oldPurchase.sku, oldPurchase.purchaseToken)
                         prorationMode?.let { prorationMode ->
                             setReplaceSkusProrationMode(prorationMode)
                         }
-                    }
+                    } ?: setObfuscatedAccountId(appUserID.sha256())
+                    // only setObfuscatedAccountId for non-upgrade/downgrades:
+                    // https://revenuecat.zendesk.com/agent/tickets/8987
```

Comment:
I think we should have a link here that's open tot he public, do we have public-facing documentation (from us or third-party) that we can point to instead?